### PR TITLE
Improve/tooltips

### PR DESCRIPTION
**Pull request by zenangst**
- [x] Rename subtitle view to tooltip view
- [x] Fix bugs related to sticky tool tips when switching between fields
- [x] Add styling for tooltips
- [x] Add tooltip to the JSON
- [x] Animate tool tip

### CODE REVIEW

**Comment by NSElvis** [comment]
Just a thought, is `message` better naming than `tooltip`?

File: Source/Cells/Text/FORMTextFieldCell.m [hunk 1]
```diff
@@ -312,10 +327,14 @@ - (void)textFormFieldDidEndEditing:(FORMTextField *)textField
         [self.textField setValid:[self.field validate]];
     }
 
-    [self.subtitleView removeFromSuperview];
+    if (self.showTooltips) {
+    [[NSNotificationCenter defaultCenter] postNotificationName:FORMDismissTooltipNotification
```
Comment (NSElvis) [inline Source/Cells/Text/FORMTextFieldCell.m]: This indentation is weird, do you mind fixing it?

**Comment by zenangst** [comment]
I dunno, I kinda like `tooltip´.

File: Source/Cells/Text/FORMTextFieldCell.m (hunk 1 above)
Comment (zenangst) [inline Source/Cells/Text/FORMTextFieldCell.m]: Fixed 4e16e2a

**Comment by NSElvis** [comment]
Makes sense that the UI component is tooltip, but in the JSON maybe it should be `message`.

What do you think @fespinoza?

**Comment by zenangst** [comment]
Yah that I can agree on :+1:

**Comment by zenangst** [comment]
The tooltip is an implementation detail but the JSON should be agnostic to that.
My vote is on `message` for the JSON

**Comment by zenangst** [comment]
@NSElvis fixed!

**Comment by NSElvis** [comment]
Great work!!! 

minion

**Pull request merged**